### ISSUE TITLE
cef, electron, and chromium crashes the device to the dev mode warning screen

### ISSUE THREAD

**Issue by honey-bbq-sauce** [dnschneid/crouton#4790]
```
name: bullseye
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/bullseye...
crouton: version 1-20220111101058~master:a9eafaee
release: bullseye
architecture: amd64
xmethod: xorg
targets: xfce
host: version 14909.100.0 (Official Build) stable-channel grunt 
kernel: Linux localhost 5.10.120-16471-g167ad2e5dcba #1 SMP PREEMPT Sat Jul 30 19:48:20 PDT 2022 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/bullseye...```

#### Please describe your issue:


#### If known, describe the steps to reproduce the issue:

**Comment by RoshiWaves** [comment]
I have the same issue

**Comment by TomTravis** [comment]
yes delete chroot fsck and start over firefox is stable i use a sd card

**Comment by dodfsk** [comment]
I have the same issue since v107

chronos@localhost / $ sudo edit-chroot -all
name: bionic
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/bionic...
/usr/local/bin/crash_reporter_wrapper: 119: exec: /var/run/crw/crw: Permission denied
WARNING: Unable to register core dump handler.
crouton: version 1-20221026163045~master:49455aeb
release: bionic
architecture: amd64
xmethod: xorg
targets: xorg,xiwi,audio,xfce,x11,extension,keyboard,core,cli-extra
host: version 15117.87.0 (Official Build) stable-channel nocturne 
kernel: Linux localhost 5.10.142-19740-g3a142da07016 #1 SMP PREEMPT Wed Oct 26 21:16:53 PDT 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/bionic...

**Comment by honey-bbq-sauce** [comment]
> yes delete chroot fsck and start over firefox is stable i use a sd card

While that does replace the web browser it doesn't fix issues with applications with chromium embedded inside them, such as steam

**Comment by dodfsk** [comment]
> > yes delete chroot fsck and start over firefox is stable i use a sd card
> 
> While that does replace the web browser it doesn't fix issues with applications with chromium embedded inside them, such as steam


Yes, I use many electron apps in crouton.vscode in crostini is so bad,crouton have higher performance.Now I choose pwa vscode instead it.
When I restored crouton to a backup from 2 months ago,it didn't fix the issue.so I gave up.

I got a kernel update from version 4.4 to 5.10 from chromeOS 107,i need a fix for the issue
Obviously,crouton is better than crostini.

**Comment by TomTravis** [comment]
post you chrome book info
it very well be another issue
chrome://system
chrome://version
![Screenshot 2022-11-06 1 30 52 PM](https://user-images.githubusercontent.com/46386261/200210405-3916a71a-5608-4125-a4d7-2d59c2bb0d69.png)
![Screenshot 2022-11-06 1 28 16 PM](https://user-images.githubusercontent.com/46386261/200210419-75c7fe05-ce47-43f7-bc35-962d4a2db705.png)

**Comment by TomTravis** [comment]
dual boot in flex is what i would Like to have

**Comment by rastshawn** [comment]
I also have this issue. Chromium and VS Code both previously worked, but after some update (not sure which) opening either totally crashes the machine back to the OS verification screen. This is on a Samsung Galaxy Chromebook.

chronos@localhost / $ sudo edit-chroot -all
name: bullseye
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/bullseye...
/usr/local/bin/crash_reporter_wrapper: 119: exec: /var/run/crw/crw: Permission denied
WARNING: Unable to register core dump handler.
crouton: version 1-20220111101058~master:a9eafaee
release: bullseye
architecture: amd64
xmethod: xorg
targets: xfce
host: version 15437.61.0 (Official Build) stable-channel hatch 
kernel: Linux localhost 5.15.108-18910-gab0e1cb584e1 #1 SMP PREEMPT Mon Jun 26 15:56:08 PDT 2023 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/bullseye...
